### PR TITLE
VDS-2631: Create new project

### PR DESCRIPTION
**Pull request by pushnitsa**

### CODE REVIEW

File: src/VirtoCommerce.DescriptionExportImportModule.Web/module.manifest [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<module>
+    <id>VirtoCommerce.DescriptionExportImport</id>
+    <version>1.0.0</version>
+    <version-tag></version-tag>
+    <platformVersion>3.23.0</platformVersion>
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Web/module.manifest:6]: Up platform version dependency to actual

File: src/VirtoCommerce.DescriptionExportImportModule.Core/VirtoCommerce.DescriptionExportImportModule.Core.csproj [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <!-- Project is not a test project -->
+        <SonarQubeTestProject>false</SonarQubeTestProject>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.23.0" />
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Core/VirtoCommerce.DescriptionExportImportModule.Core.csproj:19]: Up platform dependency

File: src/VirtoCommerce.DescriptionExportImportModule.Data/Repositories/VirtoCommerceDescriptionExportImportDbContext.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        {
+        }
+
+        protected VirtoCommerceDescriptionExportImportDbContext(DbContextOptions options)
+            : base(options)
+        {
+        }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            // modelBuilder.Entity<MyModuleEntity>().ToTable("MyModule").HasKey(x => x.Id);
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Data/Repositories/VirtoCommerceDescriptionExportImportDbContext.cs]: Remove commented code

File: src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        <Folder Include="Services\" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.8">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.23.0" />
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj:36]: Up version

File: src/VirtoCommerce.DescriptionExportImportModule.Web/Readme.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# VirtoCommerce.DescriptionExportImportModule
+
+## Overview
+
+Short overview of what the new module is.
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Web/Readme.md:5]: Add short description.

**Review by trueboroda — CHANGES_REQUESTED** [review]

**Review by trueboroda — APPROVED** [review]

**Review by ilyawzrd — APPROVED** [review]

**Pull request merged**